modules/cnv-dedicated-resources.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv-edit-vms.adoc
+// * cnv/cnv_vm_templates/cnv-editing-vm-template.adoc
+
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-dedicated-resources_{context}"]
+
+= Enabling dedicated resources for a {object}
+
+Use this procedure to enable dedicated resources for a {object}.
+
+When you enable dedicated resources for your {object},
+you obtain CPU time from a specific CPU or set of CPUs, maximizing efficiency
+while optimizing usage of system resources.
```

Comment:
hmmm, this is not entirely true. You do not optimize usage of system resources by this at all, quite a contrary. What you actually do is that you make sure that no one else will use the same resources and that the cpus of your VMs will be pinned to them.
So, something like this:
When you enable dedicated resources for your VM, your VM will be scheduled on CPUs which will not be used by any other process. This will improve the performance of the VM as well as will improve the predictability of the latency.

(please note that the templates are not scheduled anywhere, so I have changed the "{object}" to VM)

Reply:
Changed to: "When you enable dedicated resources for your virtual machine, your Virtual 
machine's workload will be scheduled on CPUs that will not be used by any other 
process, improving the performance of the virtual machine as well 
improving latency predictions."

Question @jelkosz: Good to know this doesn't affect templates. Will it ever? If not, I will strip out conditional logic from this module and remove the module's inclusion in the template assembly. Thanks! Bob

Reply:
@bgaydosrh well, not affect... The thing is that you can configure this on a template and than when you create a VM out of this template, the VM will be affected. It is the same as with any other configuration on the vm vs template. For example memory. If you set some amount of memory on the template, it does not mean that this template will consume that amount of memory. It only means that once you create a VM out of it, this VM will be consuming that amount of memory. The template is static, it never runs so it does not consume anything. It serves only as a base for the VM configuration.

Reply:
That does make a lot of sense, thanks. Let me consult with Andrew B. and see what he thinks about including DE in the template assembly.

Reply:
Yeah, we should include this in the template assembly, however it will likely need it's own para separated by conditionals to capture the difference in behaviour.
ie. 
ifdef::cnv-vm-template[]
If you enable DR for a VMT, virtual machines created from the template ...
endif::cnv-vm-template[]
ifdef::cnv-vm[]
If you enable DR for a VM, the VM ...
endif::cnv-vm[]

Reply:
I added conds for the opening sentence in the procedure to indicate the slight difference in template/non-template functionality.